features/rollershutter_item.feature

```diff
@@ -0,0 +1,71 @@
+Feature: rollershutter_item
+  Rollershutter Items are supported
+
+  Background:
+    Given Clean OpenHAB with latest Ruby Libraries
+    And group "Rollershutters"
+    And items:
+      | type          | name      | label        | group          |
+      | Rollershutter | RollerOne | Roller One   | Rollershutters |
+      | Rollershutter | RollerTwo | Roller Two   | Rollershutters |
+
+  Scenario: Sending command using PercentType work
+    Given item "RollerOne" state is changed to "<initial>"
+    And item "RollerTwo" state is changed to "0"
+    And code in a rules file
+      """
+      java_import Java::OrgOpenhabCoreLibraryTypes::DecimalType
+
+      RollerOne << <command>
+      """
+    When I deploy the rules file
+    Then "RollerOne" should be in state "<final>" within 5 seconds
+      Examples:
+          | initial | command             | final |
+          | 50      | 70                  | 70    |
+          | 50      | RollerTwo           | 0     |
+          | 50      | DecimalType.new(10) | 10    |
+
+  Scenario: Command methods work
+    Given item "RollerOne" state is changed to "<initial>"
+    And code in a rules file
+      """
+      RollerOne.<method>
+      """
+    When I deploy the rules file
+    Then "RollerOne" should be in state "<final>" within 5 seconds
+      Examples:
+          | initial | method | final |
+          | 50      | up     | 0     |
+          | 50      | down   | 100   |
+
+  Scenario: Comparisons work
+    Given item "RollerOne" state is changed to "50"
+    And item "RollerTwo" state is changed to "0"
+    And code in a rules file
+      """
+      logger.info(RollerOne <operator> <operand>)
+      """
+    When I deploy the rules file
+    Then It should log "<result>" within 5 seconds
+      Examples:
+          | operator | operand    | result   |
+          | ==       | 50         | true     |
+          | >        | RollerTwo  | true     |
+
+  Scenario: up? and down? methods work
+    Given item "RollerOne" state is changed to "<initial>"
+    And code in a rules file
+      """
+      logger.info(RollerOne.<method>)
+      """
+    When I deploy the rules file
+    Then It should log "<result>" within 5 seconds
+      Examples:
+          | initial | method | result |
+          | 0       | up?    | true   |
+          | 0       | down?  | false  |
+          | 100     | up?    | false  |
+          | 100     | down?  | true   |
+          | 50      | up?    | false  |
+          | 50      | down?  | false  |
```

Comment:
Once it extends numeric we should ensure that case statements work..
```
when RollerShutter
case UP:
case Down:
case 50:
case 0..25: 
```

Reply:
That's a great idea, would just need to ovverride `===` for that, right?

Reply:
Need to make it implement numeric and coerce so that '0..25' or `50` work.   Need to override === on UP and DOWN for the other to work.   Look at the monkeypatch on on_off_type.rb.

Reply:
Added a monkey-patch to UpDownType so it supports case equality.